fork/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -367,8 +367,8 @@ $container-lg:                 $container-large-desktop !default;
 $navbar-height:                    50px !default;
 $navbar-margin-bottom:             $line-height-computed !default;
 $navbar-border-radius:             $border-radius-base !default;
-$navbar-padding-horizontal:        floor(math.div($grid-gutter-width, 2)) !default;
-$navbar-padding-vertical:          math.div($navbar-height - $line-height-computed, 2) !default;
+$navbar-padding-horizontal:        floor(calc(#{$grid-gutter-width} / 2)) !default;
```

Comment:
floor + calc do not work, just use calc instaed

Reply:
thanks, floor/ceil removed